### ISSUE TITLE
Azure AD non B2C

### ISSUE THREAD

**Issue by Vayth** [moodio/msal-flutter#14]
Can I use this for Azure AD for organization? I'm having difficulties figuring the authorities since I always got null client/authority

**Comment by Sarthak261995** [comment]
I don't think this package provide MSAL authentication(Non B2C) I don't see any example for that in samples.
